src/pages/certifications/javascript-algorithms-and-data-structures/basic-javascript/testing-objects-for-properties/index.md

```diff
@@ -3,8 +3,21 @@ title: Testing Objects for Properties
 ---
 ## Testing Objects for Properties
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/javascript-algorithms-and-data-structures/basic-javascript/testing-objects-for-properties/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+My changes:
```

Comment:
Please replace `My Changes:` with a level 2 heading `Testing Objects for Properties`.

Level 2 headings are 2 hashtags `##`.